app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,16 @@
+<footer class="footer">
+  <small>
+    <%= t(:translate, :item=> t('translates.the')) %>
+      <%= link_to t('RORT'), "https://www.railstutorial.org/" %>
+    <%= t(:translate, :item=> t('translates.by'))%>
```

Comment:
`:item=>` -> `item:`

`t('translates.the')` -> nháy đôi

self check file này